src/controller/python/chip/ChipStack.py

```diff
@@ -174,6 +175,7 @@ def HandleChipThreadRun(callback):
         im.InitIMDelegate()
         ClusterAttribute.Init()
         ClusterCommand.Init()
+        Bdx.Init()
```

Comment:
Is there a way to limit this only to test builds?

Reply:
I think it's okay to leave this here. The intent is to expand this into a full BDX implementation that can be used outside of tests. This lets people use the partial implementation if they want. The Init method only sets up the C/Python function mapping, so I don't think it will be an issue to include it.